pages/status.tsx

```diff
@@ -112,15 +112,26 @@ const Status: FC = () => {
                 goBackStyle="primary"
                 checkAgainText={t('check-again')}
               >
-                <p className="mb-6 text-2xl">
-                  {`${t('status-is')} `}
-                  <strong id="response-status">
-                    {t(`status.${checkStatusReponse.status}`, {
-                      defaultValue: checkStatusReponse.status,
-                    })}
-                  </strong>
-                  .
-                </p>
+                <div>
+                  <p className="mb-6 text-2xl">
+                    {`${t('status-is')} `}
+                    <strong id="response-status">
+                      {t(`status.${checkStatusReponse.status}.label`, {
```

Comment:
Remove `defaultValue` option. I prefer to display `status.99,label` if the key is missing. I know that I am the one who added  `defaultValue` at the beginning. 😄